v1/migration/index.php

```diff
@@ -30,6 +30,11 @@
 		'build'   => $build_url,
 		'version' => $version,
 	],
+	'php' => [
+		'min' => '5.6',
+		'max' => '7.4.x',
+		'unsupported' => '8.0',
```

Comment:
I'm not sure that we need all three of these version numbers. Wouldn't `min => '5.6'` and `max => '7.4.99'` be sufficient?

Reply:
The 'max' version is directly reported to users so notifying users that we support as yet unreleased versions of PHP may look a little strange.

Perhaps I can alter the plugin to drop the patch segment of the version and report it as `7.4.x` or just `7.4` instead.

Thoughts?

Reply:
Or perhaps even better:

```
'min' => '5.6',
'max' => '7.4.999',
'max_display' => '7.4.x',
```

Reply:
To put it another way, I guess having both `max` and `unsupported` doesn't make sense to me, since this is basically the same data point but repeated in a different way.

Reply:
Agreed, that's a better approach - PR updated.